demos/simple/index.html

```diff
@@ -23,7 +23,104 @@
 </head>
 <body>
     <h1>Live Editor Example</h1>
-    <div id="sample-live-editor"></div>
+    <div id="sample-live-editor">
+        <div class="scratchpad-wrap">
+            <!-- Canvases (Drawing + Output) -->
+            <div class="scratchpad-canvas-wrap">
+                <div id="output">
+                    <!-- Extra data-src attribute to work around
+                         cross-origin access policies. -->
+                    <iframe id="output-frame"
+                            src="output.html"
+                            data-src="output.html"></iframe>
+                    <canvas class="scratchpad-draw-canvas" style="display:none;"
+                            width="400" height="400"></canvas>
+
+                    <div class="overlay disable-overlay" style="display:none;">
+                    </div>
+
+                    <div class="scratchpad-canvas-loading">
+                        <!--<img src="../../images/throbber-full.gif">-->
+                        <!--<span class="hide-text">{{#_}}Loading...{{/_}}</span>-->
+                    </div>
+                </div>
+
+                <div class="scratchpad-toolbar">
+                    <button id="restart-code"
+                            class="simple-button pull-right">
+                        <span class="icon-refresh"></span>
+                        Restart
+                    </button>
+
+                    <!-- Widgets for selecting colors to doodle on the canvas during
+                        recordings -->
+                    <!--<div id="draw-widgets" style="display:none;">-->
```

Comment:
I commented out the stuff for making recordings.  It would be nice if this HTML (and associated JavaScript) was dynamically loaded only when making recordings.